.github/workflows/cd.yml

```diff
@@ -0,0 +1,37 @@
+name: CD
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Set output
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Set up JDK 8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Build with Gradle
+        run: ./gradlew clean build
+        env:
+          CTP_PROJECT_CONFIG: ${{ secrets.CTP_PROJECT_CONFIG }}
```

Comment:
I think these Environment variables are wrong.
The correct once`s are:
- IT_CLIENT_ID  
- IT_CLIENT_SECRET  
- IT_PROJECT_KEY  (payment-to-order-processor-7)

Reply:
https://github.com/commercetools/commercetools-payment-to-order-processor/pull/85/commits/75921e89c1b9141e3bbfad10bf6c21c9f204c5fd